doc/snacks-dashboard.txt

```diff
@@ -110,33 +110,22 @@ In the example below, both sections are equivalent.
 
 >lua
     ---@class snacks.dashboard.Config
-    ---@field sections (snacks.dashboard.Section|fun(opts:snacks.dashboard.Opts):(snacks.dashboard.Section|snacks.dashboard.Section[]|nil))[]
-    ---@field formats table<string, snacks.dashboard.Text|fun(value:string):snacks.dashboard.Text>
+    ---@field sections snacks.dashboard.Section
+    ---@field formats table<string, snacks.dashboard.Text|fun(item:snacks.dashboard.Item, ctx:sncaks.dashboard.Format.ctx):snacks.dashboard.Text>
```

Comment:
Minor typo: `ctx:snacks.dashboard.Format.ctx` here and the other files as well?

Reply:
Yep indeed. Just pushed a big refactor :)